deps/ncrypto/ncrypto.cc

```diff
@@ -311,6 +311,83 @@ BignumPointer BignumPointer::clone() {
   return BignumPointer(BN_dup(bn_.get()));
 }
 
+int BignumPointer::isPrime(int nchecks,
+                           BignumPointer::PrimeCheckCallback innerCb) const {
+  BignumCtxPointer ctx(BN_CTX_new());
+  BignumGenCallbackPointer cb(nullptr);
+  if (innerCb != nullptr) {
+    cb = BignumGenCallbackPointer(BN_GENCB_new());
+    if (!cb) [[unlikely]]
+      return -1;
+    BN_GENCB_set(
+        cb.get(),
+        [](int a, int b, BN_GENCB* ctx) mutable -> int {
```

Comment:
This will never get inlined this way. Can we move this to a function or at least add a TODO here?